### ISSUE TITLE
Order emails partially translated even after #401

### ISSUE THREAD

**Issue by Annih** [hyyan/woo-poly-integration#415]
#### Can you reproduce this issue on default Wordpress theme (eg Storefront)?
Yes

#### Can you reproduce this issue when all other plugins are disabled except WooCommerce, Polylang and Hyyan WooCommerce Polylang Integration?
Can't test that at the moment

#### What product versions and settings are you using when this issue occurs?
* PHP: 7.2.17
* WordPress: 5.1.1
* WooCommerce: 3.6.2
* Polylang: 2.5.3
* Hyyan WooCommerce Polylang Integration: 
* Browser: 1.3.0

#### Steps to Reproduce
1. Setup 2 languages
2. Customize order emails subject and  heading
3. Place an order in the secondary language

#### What I Expected
The whole mail (subject, heading, body) to be translated.

#### What Happened Instead
Body was properly translated, not the subject nor the heading.

**Comment by Annih** [comment]
This is a variant of #401 still active after the fix (89ca15df970ee410513200954f0993fcf0ab93f9).

During my investigation I discovered that the call to `get_locale` (https://github.com/hyyan/woo-poly-integration/commit/89ca15df970ee410513200954f0993fcf0ab93f9#diff-69fcb943770f73661e1cb4d114de1460R681) was sometimes returning fr_FR and sometimes en_US ***for the same order***.

I first decided to use the global `WP_LANG` locale to ensure that the comparison with the order locale was done with the site locale. (https://github.com/hyyan/woo-poly-integration/commit/89ca15df970ee410513200954f0993fcf0ab93f9#diff-69fcb943770f73661e1cb4d114de1460R687)
As pointed by @Jon007  this attempt was not the best thing to do. I made it because I thought the locale comparison was only conditioning the translation https://github.com/hyyan/woo-poly-integration/commit/89ca15df970ee410513200954f0993fcf0ab93f9#diff-69fcb943770f73661e1cb4d114de1460R698
But it is also loading the correct locale https://github.com/hyyan/woo-poly-integration/commit/89ca15df970ee410513200954f0993fcf0ab93f9#diff-69fcb943770f73661e1cb4d114de1460R696

Now that I read the code multiple times, I think that 89ca15df970ee410513200954f0993fcf0ab93f9 introduced a new bug: "when the locale is already loaded, translation is not performed"
I think we just need to move the translation call outside the "switchLocale condition branch".
